AWESOME.md

```diff
@@ -36,3 +41,177 @@
+- [What is Holochain's consensus algorithm?](https://developer.holochain.org/guide/latest/faq.html#what-is-holochains-consensus-algorithm)
+- [Can you run a cryptocurrency on Holochain?](https://developer.holochain.org/guide/latest/faq.html#can-you-run-a-cryptocurrency-on-holochain)
+- [How is Holochain different from __________?](https://developer.holochain.org/guide/latest/faq.html#how-is-holochain-different-from-__________)
+- [What language is Holochain written in? What languages can I use to make Holochain apps?](https://developer.holochain.org/guide/latest/faq.html#what-language-is-holochain-written-in-what-languages-can-i-use-to-make-holochain-apps)
+- [Is Holochain open source?](https://developer.holochain.org/guide/latest/faq.html#is-holochain-open-source)
+- [How is Holochain more environmentally ethical than blockchain?](https://developer.holochain.org/guide/latest/faq.html#how-is-holochain-more-environmentally-ethical-than-blockchain)
+- [How are data validated on Holochain?](https://developer.holochain.org/guide/latest/faq.html#how-are-data-validated-on-holochain)
+- [What happens to data when a node leaves the network?](https://developer.holochain.org/guide/latest/faq.html#what-happens-to-data-when-a-node-leaves-the-network)
+- [Should I build my coin/token on Holochain?](https://developer.holochain.org/guide/latest/faq.html#should-i-build-my-cointoken-on-holochain)
+- [What does “agent-centric” mean? How is this different from “data-centric”?](https://developer.holochain.org/guide/latest/faq.html#what-does-agent-centric-mean-how-is-this-different-from-data-centric)
+- [What is the TPS (Transactions Per Second) on Holochain?)](https://developer.holochain.org/guide/latest/faq.html#what-is-the-tps-transactions-per-second-on-holochain)
+- [Holo Fuel Economics 101](https://medium.com/h-o-l-o/holo-fuel-economics-101-c9631d63014a)
+- [Holo: The evolution of cloud computing](https://medium.com/h-o-l-o/holo-the-evolution-of-cloud-computing-57a0a281b88b)
+- [Holo’s Value](https://medium.com/h-o-l-o/holos-value-91888d24121)
+- [Holo’s Org-Wide Progress in 2018](https://medium.com/h-o-l-o/holos-org-wide-progress-in-2018-40c71361bedf)
+- [Holochain : The Blockchain picks up a Dimension](https://medium.com/holochain/holochain-the-blockchain-picks-up-a-dimension-4dca43fe3a73)
+- [The Legal Principles of the Cryptographic Autonomy License (CAL)](https://medium.com/h-o-l-o/some-legal-aspects-of-the-cryptographic-autonomy-license-cal-c95e18c90388)
+- [Junto: A New Breed of Social Media Powered by Holochain](https://medium.com/holochain/junto-a-new-breed-of-social-media-powered-by-holochain-1faeff487f1d)
+- [Holochain: Reinventing Applications](https://medium.com/holochain/holochain-reinventing-applications-d2ac1e4f25ef)
+
+#### Developer Kit Resources
+
+- [Crate HDK](https://developer.holochain.org/api/0.0.2/hdk/index.html)
```

Comment:
maybe go for 0.0.3 version for now, since it's the latest?
```suggestion
- [Crate HDK](https://developer.holochain.org/api/0.0.3/hdk/index.html)
```